opentreemap/importer/models/base.py

```diff
@@ -30,6 +31,15 @@ class Meta:
     FAILED_FILE_VERIFICATION = 6
     CANCELED = 8
     VERIFICATION_ERROR = 10
+    UNKNOWN_ERROR = 12
+
+    @classproperty
+    def INACTIVE_CODES(self):
```

Comment:
If we adopt OpenTreeMap/otm-addons#1045 I think this variable will no longer have two uses, and we should:
* add the new statuses to `is_finished` (possibly renaming it...)
* change the query for what goes in the "Finished" table to include only `FINISHED_CREATING` and `FAILED_FILE_VERIFICATION`. (`VERIFICATION_ERROR` indicates an internal error, so the job should be resumable.)